atomic_reactor/plugins/fetch_sources.py

```diff
@@ -683,9 +683,11 @@ def get_srpm_urls(self, sigkeys=None, insecure=False):
             self.log.debug('Resolving SRPM for RPM ID: %s', rpm_id)
 
             if rpm['external_repo_name'] != 'INTERNAL':
-                msg = ('RPM comes from an external repo (RPM ID: {}). '
-                       'External RPMs are currently not supported.').format(rpm_id)
-                raise RuntimeError(msg)
+                msg = ('RPM comes from an external repo (RPM ID: {}; NVR: {}). '
+                       'External RPMs are currently not supported, '
+                       'skipping').format(rpm_id, rpm['nvr'])
```

Comment:
what are we skipping here?

Reply:
when I was testing some konflux image, they had some RPMs from external repo, so for that we can't get sources